docs/getting_started.md

```diff
@@ -141,15 +148,17 @@ You can check the status of your configuration with:
 ```
 
 ### Verifying the AWS Adapter installation
-Perform the below steps to see if the adapter is installed correctly
+
+Perform the below steps to see if the adapter is installed correctly.
+
 ```bash
-# check if the controller pod is running
+# check if the controller pod is running.
 kubectl get pods -n nirmata-aws-adapter
-NAME                                              READY   STATUS    RESTARTS   AGE
-aws-adapter-kyverno-aws-adapter-997f45bb9-c2z5j   1/1     Running   0          109m
+NAME                                  READY   STATUS    RESTARTS   AGE
+kyverno-aws-adapter-997f45bb9-c2z5j   1/1     Running   0          109m
 
-# check the awsadapterconfig status
+# check the awsadapterconfig status.
 kubectl get awsacfg -n nirmata-aws-adapter
-NAME                             ...   CLUSTER NAME     ...  STATUS
-aws-adapter-kyverno-aws-adapter  ...   cluster-name     ...  success
+NAME                 ...   CLUSTER NAME     ...  LAST POLLED STATUS
```

Comment:
I thought `Last Polled` was supposed to indicate the time at which it was last polled

Reply:
This is what we have right now,
![image](https://user-images.githubusercontent.com/125429069/220933665-0248a306-56fd-4130-a56b-2b069e819b1e.png)

Reply:
Oh I thought those were 2 different columns